docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -0,0 +1,153 @@
+---
+id: enhanced-project-workflow
+title: Enhanced Project Workflow
+sidebar_label: Enhanced Project Workflow
+---
+
+:::caution
+This is a **BETA** feature and, as such, is currently under active development. Pay attention using it.
+:::
+
+The **Enhanced Project Workflow** feature aims to improve the developer experience when working with the Console, as well as pave the way for the adoption of previously unavailable opportunities.
+
+## Changes to the Console workflow
+
+The following sections will highlight the changes made to the established Console workflow, regarding in particular the new management of the project logical state, and the new deployment workflow.
+
+### New configuration management strategy
+
+The most relevant change in the Console’s behavior is that the project’s logical state will not be saved on a Git provider anymore. This will result in significant **performance improvement** during the main Console workflows, since contacting the Git provider implied a considerable overhead that will now be reduced.
+
+This change also means that the Console will not rely on the typical Git-based instruments anymore and will instead adopt a **new configuration management system**, while still maintaining consistency with the Console’s established workflows.
+
+With the new workflow, branches and tags become **revisions** and **versions**, underscoring that the information associated with them is not stored within a Git repository anymore. The concept of commit is not present anymore and the commit history is replaced by a **version history**: the user can create versions, similar to Git tags, to keep track of the project’s evolution.
```

Comment:
Proposal to make it a little more concise and clear. Avoiding to literal translation of italian terms like `underscoring` that sounds odd

```suggestion
To emphasize that the new workflow configuration is no longer stored in a Git repository:
- `branches` and `tags` are now **`revisions`** and **`versions`**
- `commit history` is now **`version history`**

To keep track of the project’s evolution, user can create `versions` (similar to Git tags) that act as fixed snapshots of the configuration in a particular moment.
```